LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Daniel Connelly
```

Comment:
Will you please add Teal Dulcet as well.

Reply:
@ tdulcet,

License has been added, please let me know if you have any questions.